### PR TITLE
FORMS-9278: Fix repeatability style in WKND

### PR DESCRIPTION
**Pull request by akshayvas**
## Screenshots (if appropriate):

image (9)
image (8)
image (7)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by akshayvas** [comment]
@amit-mnnit @suryansh29 Please review

File: src/site/_variables.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@ $gray: rgb(109, 109, 109);
 $light-gray: #969696;
 $very-light-gray: #ddd;
 $disabled-color: $background-color;
+$bg-gray: #F1F1F1;
```
Comment (amit-mnnit) [inline src/site/_variables.scss:16]: Don’t create new variable

File: src/components/accordion/_accordion.scss [hunk 2]
```diff
@@ -67,6 +67,34 @@
     }
   }
 
+  &__header{
+    display: flex;
+  }
+  &__button{
+    flex: 1;
+  }
+  &__repeatable-buttons{
```
Comment (amit-mnnit) [inline src/components/accordion/_accordion.scss:76]: create mixin for this

**Review by amit-mnnit — COMMENTED** [review]
Please don’t create new variable

**Comment by akshayvas** [comment]
@amit-mnnit addressed review comments, please review

**Review by amit-mnnit — APPROVED** [review]

**Pull request merged**